net/goai/goai_shema.go

```diff
@@ -204,6 +205,61 @@ func (oai *OpenApiV3) structToSchema(object interface{}) (*Schema, error) {
 			if validationRuleSet.Contains(validationRuleKeyForRequired) {
 				schema.Required = append(schema.Required, key)
 			}
+
+			// Extract validation rules to schema. like min, max, length
+			validationRules := ref.Value.ValidationRules
+			// remove custom error message
+			if idx := strings.Index(ref.Value.ValidationRules, "#"); idx != -1 {
```

Comment:
You can refer current parsing logic in package `goai`, which uses `gvalid.ParseTagValue` to parse the validation pattern: https://github.com/gogf/gf/blob/e8a2629b19ddbd4144bbd69533ef4e6dd00ace5e/net/goai/goai_shema.go#L230

Reply:
up to this point, gvalid.ParseTagValue(validationTagValue) has already been processed. The rules have already removed the custom error messages, and strings.index is no longer needed.

Reply:
it is indeed.